examples/voltage_in_ratio.rs

```diff
@@ -0,0 +1,140 @@
+// phidget-rs/examples/voltage_in.rs
+//
+// Copyright (c) 2023, Frank Pagliughi
+// implemented by Jorge Guerra and Riley Hernandez 2024
+//
+// This file is an example application for the 'phidget-rs' library.
+//
+// Licensed under the MIT license:
+//   <LICENSE or http://opensource.org/licenses/MIT>
+// This file may not be copied, modified, or distributed except according
+// to those terms.
+//
+
+//! Rust Phidget example application to read voltage input values.
+//!
+//! A number of sensors have a simple, linear, formula to get a value in
+//! the form:
+//!
+//! ```text
+//! value = (voltage - offset) * gain
+//! ```
+//! This example app allows you to enter the gain and offset to see the value
+//! in the units of the sensor. For example, the [VCP4114.0 Clip-On Current
+//! Transducer](https://www.phidgets.com/?&prodid=1184) has the formula:
+//!
+//! ```text
+//! DC Amps = (V - 2.5) * 16.0
+//! ```
+//!
+//! So, to see the output in amps, run this:
+//!
+//! ```text
+//! $ voltage_in -o 2.5 -g 16.0
+//! ```
+//!
+//! The input bit can be selected by choosing the serial number of a device
+//! and channel number for the input.
+//!
+//! You can also use a port on a hub as a voltage intput. In that case the
+//! voltage is measured between the white line (signal) and black line
+//! (ground). Select the hub (-h) option and the port number, like:
+//!
+//! ```text
+//! $ voltage_in -h -p 5
+//! ```
```

Comment:
^^^ This is the text/comment from the `voltage_in.rs` file?

Please update it to explain what this new examples does and how it works.